include/irr/asset/ICPUDescriptorSet.h

```diff
@@ -148,6 +151,10 @@ class ICPUDescriptorSet final : public IDescriptorSet<ICPUDescriptorSetLayout>,
 		//! Can modify the array of descriptors bound to a particular bindings
 		inline core::SRange<SDescriptorInfo> getDescriptors(uint32_t index) 
 		{ 
+			//this must be commented-out, otherwise cpu2gpu doesnt have access to descriptors
+			//if (isImmutable_debug())
+			//	return core::SRange<SDescriptorInfo>{nullptr, nullptr};
```

Comment:
does the cpu2gpu need non-`const` `SDescriptorInfo` ?

Reply:
need non-const descriptors

Reply:
but why?